capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -52,6 +52,12 @@ class ClubPage extends Component {
 
     this.setState({ club });
 
+    if (!club.memberIDs.includes(window.localStorage.getItem('userID')) 
+        && club.ownerID !== window.localStorage.getItem('userID')) {
```

Comment:
I think the more expensive check here would be the `.includes` would it be possible to move the owner check to the beginning in place of the member `includes` check. This would help increase short circuiting and remove unnecessary computation

Reply:
Good idea! I think the check is currently fairly quick so this wouldn't be a huge improvement, but I think that it's a good idea and would save more time as club size increases. I'll change this.